src/analysis/bounds.rs

```diff
@@ -203,16 +205,21 @@ impl Bounds {
             }) => Some(IsA(None)),
             Type::Interface(..) => Some(IsA(None)),
             Type::List(_) | Type::SList(_) | Type::CArray(_) => None,
-            Type::Fundamental(_) if *nullable => None,
             Type::Function(_) => Some(NoWrapper),
             _ => None,
         }
     }
 
-    fn get_to_glib_extra(bound_type: &BoundType) -> String {
+    fn get_to_glib_extra(
+        bound_type: &BoundType,
+        nullable: bool,
+        instance_parameter: bool,
+    ) -> String {
```

Comment:
I wonder why we don't return a `&'static str` instead... We don't format anything after all...

Reply:
For allowing more complicated things later. We had that here at some point in the past.

Changing the type would require changing it in lots of other places too.